src/components/settings/SettingsSetLogLevel.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+	<div>
+		<fieldset>
```

Comment:
Do we need this nesting? Maybe we could just drop the `div` and use the `fieldset` as root?

Reply:
Done